### ISSUE TITLE
Treasury: Table index is nil

### ISSUE THREAD

**Issue by Shylo-Musgrove** [Windower/Lua#2063]
I kept getting the error "Table index is nil". After about 1.5 hours of troubleshooting I found it was due to the WHM body renaming that SE did in the June update. They changed all versions of them from Briault to Bliaut. Once I deleted all of the WHM bodies (to include the -1) the issue was resolved.

I would either update Treasury to auto-update the names in everyone's settings file or just remove them and let the user re-add with the updated name.

**Comment by paulframe** [comment]
The -1 whm relic body was NOT renamed
https://www.bg-wiki.com/ffxi/Clr._Briault_-1

**Comment by RubenatorX** [comment]
The -1 whm relic body was __not__ renamed
https://www.bg-wiki.com/ffxi/Clr._Briault_-1
https://www.bg-wiki.com/ffxi/June_2021_Version_Update_Changes/Existing_Items

**Comment by Shylo-Musgrove** [comment]
> The -1 whm relic body was **not** renamed
> https://www.bg-wiki.com/ffxi/Clr._Briault_-1
> https://www.bg-wiki.com/ffxi/June_2021_Version_Update_Changes/Existing_Items

Ah ok. I did not individually test that. I did a find for Briault and deleted all of them for WHM, including the -1, and I stopped receiving the error.

**Comment by Shylo-Musgrove** [comment]
Suggestion while I have this open and it is related...I had the same error on another account. Instead of "harness" it had "harn". This was likely caused by me from troubleshooting. Not sure if I deleted a "." or the "ess". I changed it to harness and the error is removed.

Is there a way the error report we get in console can be more specific? If it told me it didn't recognize the item name "harn", I could have just searched that and fixed the issue in mere minutes. Instead, I spent 30 minutes troubleshooting until I was able to isolate the issue. Lots of comment these out, reload, still error, comment more out, reload, etc etc. I had over 500 items listed in this characters <Pass \>.

Another option would be to ignore unknown names (still prompt error in console). Nothing is being dropped or passed or lotted by ignoring the incorrect name. The "table index is null" error rendered treasury useless until I corrected the errors.
